common/src/main/java/com/datastrato/gravitino/utils/MapUtils.java

```diff
@@ -10,6 +10,11 @@
 import java.util.Map;
 
 public class MapUtils {
+  // Add private constructor to make the class non-instantiable
+  private MapUtils() {
+    throw new AssertionError("MapUtils class should not be instantiated.");
```

Comment:
We typically don't throw an exception here. just empty constructor is enough. like this:

```java
private MapUtils() {}
```

Also you can check the CI result to fix the style issue.